helm_deploy/apply-for-legal-aid/templates/_helpers.tpl

```diff
@@ -46,3 +46,19 @@ https://pauladamsmith.com/blog/2011/05/go_time.html
     {{ "* * * * *" }}
   {{- end -}}
 {{- end -}}
+
+{{/*
+Function to return the name for a UAT redis chart master node host
+If redis chart release name contains chart name it will be used as a full name.
```

Comment:
If branch name contains "redis" the release name only appends "master", instead of "redis-master"